Code/GraphMol/Atom.cpp

```diff
@@ -38,31 +38,130 @@ bool isAromaticAtom(const Atom &atom) {
   return false;
 }
 
-// Determine whether or not a molecule is to the left of Carbon
+// Determine whether or not an element is to the left of carbon.
 bool isEarlyAtom(int atomicNum) {
-  if (atomicNum <= 1) {
-    return false;
-  }
-  switch (PeriodicTable::getTable()->getNouterElecs(atomicNum)) {
-    case 1:
-    case 2:
-    case 3:
-      return true;
-    case 4:
-      // we make an arbitrary decision that Ge, Sn, and Pb
-      // are treated like early elements (part of github #2606)
-      return atomicNum > 14;
-    case 5:
-      // we make an arbitrary decision that Sb and Bi
-      // are treated like early elements (part of github #2775)
-      return atomicNum > 33;
-    case 6:
-    case 7:
-    case 8:
-      return false;
-    default:
-      return false;
-  }
+  static const bool table[119] = {
+    false, // #0 *
+    false, // #1 H
+    true,  // #2 He
```

Comment:
This is what the current code does, but it's wrong.
```suggestion
    false,  // #2 He
```